aim/web/ui/src/components/kit_v2/Slider/Slider.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import _ from 'lodash-es';
+
+import Text from '../Text';
+
+import { ISliderProps } from './Slider.d';
+import {
+  SliderLabel,
+  SliderMark,
+  SliderRange,
+  SliderRoot,
+  SliderThumb,
+  SliderTrack,
+} from './Slider.style';
+
+//TODO: Fix the thumb label overlapping issue
+
+const Slider = React.memo(
+  ({
+    min = 0,
+    max = 100,
+    value,
+    defaultValue = [10],
+    marks = [],
+    onValueChange,
+    ...props
+  }: ISliderProps) => {
+    const [sliderValue, setSliderValue] =
+      React.useState<number[]>(defaultValue);
+
+    React.useEffect(() => {
+      if (value) {
+        if (value && !_.isEqual(value, sliderValue)) {
+          setSliderValue(value);
+        }
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [value]);
+
+    const onChange = (value: number[]) => {
+      setSliderValue(value);
+      if (onValueChange) {
+        onValueChange(value);
+      }
+    };
+
+    function getMarkPosition(mark: number) {
+      // The mark position is calculated based on the min and max values
+      const markPosition = ((mark - min) / (max - min)) * 100;
+
+      // The coefficient is needed to align the mark with the thumb
+      const calcCoif: number = 5 - markPosition * 0.1;
+      return `calc(${markPosition}% + ${calcCoif}px)`;
+    }
+
+    return (
+      <SliderRoot
+        value={sliderValue}
+        defaultValue={defaultValue}
+        min={min}
+        max={max}
+        onValueChange={onChange}
+        data-testid='slider'
+        {...props}
+      >
+        <SliderTrack className='SliderTrack'>
+          <SliderRange className='SliderRange' />
+        </SliderTrack>
+        {sliderValue.map((_: number, i: number) => (
+          <SliderThumb className='SliderThumb' key={i}>
+            <SliderLabel className='SliderLabel'>{sliderValue[i]}</SliderLabel>
+          </SliderThumb>
+        ))}
+
+        {marks.length > 0 &&
+          marks?.map((mark) => {
```

Comment:
In case of incorrect `marks` value (undefined, null, etc...) this condition wouldn’t work and will break the app.

let's fix the condition: 
from: `marks.length > 0 && marks?.map((mark) => ...`
to: `marks?.length > 0 && marks.map((mark) => ...`